### ISSUE TITLE
create ggplot theme function/s using IPC colours and design language

### ISSUE THREAD

**Issue by ernestguevarra** [nutriverse/ipctools#11]

**Comment by ernestguevarra** [comment]
font: Myriad Pro and Myriad Pro Light
font colour: 

Acute food insecurity colours:
Minimal
C: 19 | M: 0 | Y: 26| K: 0
R: 205 | G: 250 | B: 205
Web Hex: CDFACD
Stressed
C: 4 | M: 3 | Y: 96 | K: 0
R: 250 | G: 230 | B: 30
Web Hex: FAE61E
Crisis
C: 7 | M: 63 | Y: 100 | K: 3
R: 230 | G: 120 | B: 000
Web Hex: E67800
Emergency
C: 13 | M: 100 | Y: 100 | K: 5
R: 200 | G: 000 | B: 000
Web Hex: C80000
Famine
C: 34 | M: 98 | Y: 95 | K: 54
R: 100 | G: 000 | B: 000
Web Hex: 640000

Chronic food insecurity colours:


Acute malnutrition colours:

Acceptable
C: 19 | M: 0 | Y: 26| K: 0
R: 205 | G: 250 | B: 205
Web Hex: CDFACD
Alert
C: 4 | M: 3 | Y: 96 | K: 0
R: 250 | G: 230 | B: 30
Web Hex: FAE61E
Serious
C: 7 | M: 63 | Y: 100 | K: 3
R: 230 | G: 120 | B: 000
Web Hex: E67800
Critical
C: 13 | M: 100 | Y: 100 | K: 5
R: 200 | G: 000 | B: 000
Web Hex: C80000
Extremely Critical
C: 34 | M: 98 | Y: 95 | K: 54
R: 100 | G: 000 | B: 000
Web Hex: 640000

**Comment by tomaszaba** [comment]
Hi @ernestguevarra, what would be the utility of this ggplot function/s? I am guessing to color code using the IPC colors?
 
Let me share a bit of overview of how the IPC works. The phase classification is agreed during an analysis workshop with multiple stakeholders, through consensus and using a reference table. That means, before the workshop, there should only be output tables for each domain. The output tables are then used to be consulted while filling an IPC analysis tool - basically a standardized tool where analysts go through in systematic way to populate it, then critically review the evidence. Under this definition, the brand codes are used after this workshop gets finished, while writing up what we call communication brief. 

So, in summary, this package we're creating should only be able to have functions that takes data frames turns into summarized results in way we did before. That file will then be used to feed into the IPC analysis as I explained before.

**Comment by tomaszaba** [comment]
@ernestguevarra , I would like to let you know that after our call on Monday, I had a meeting with my IPC colleagues (including one of my supervisors in charge of technical development) where I walked them through what that muac-quality-check project does and how good that is in enhancing the work we do with countries. I explained that this initiative was started as a personal interest to avoiding "suffering" again whenever I am assigned a tasked to perform data quality checks and summarize prevalences, etc. at different domains, and I highlighted that code will be public anyway.  I mentioned that we are planning to create an R package so R users can benefit from these utilities for their quality-checks.

The initiative was absolutely very well welcomed as it aggregates value in fast-tracking our tech support to IPC countries and reduce the time spent running analysis - which is also prone to errors and they already started suggesting some enhancements as described below: 

There are new developments on checking the quality of muac that we should consider including in this package, which was also proposed to do so by my supervisor. Here is a small background: In November last year, there was Global Assessment meeting held in Nairobi where Dr. Oleg presented on a data mining he did exploring measurement quality of muac  data. The authors proposed to use muac z-score. With this, it would it becomes possible to generate a SMART-like plausibility report for MUAC. Currently, the SMART team are working on updating the ENA software. So, my supervisor suggested that we could enhance the package with this new development using the same cut-off points that the SMART team will apply in the revised version of ENA. Those cut-offs are not available yet. Here is the paper: [https://onlinelibrary.wiley.com/doi/10.1111/mcn.13478](url)

On this last point, my suggestion is the following: 
- even though we do not have the cut-off points, we could probably start working on the code assuming the same cut-offs of WHZ and then once MUAC-specific cut-offs get released, we can simply update where applicable. What do you think?

Finally, I also think we could include wasting caseload function in this package, what do you think? My rational is that the `ipctools` would have functions/utilities for IPC related data analysis. 

Let me know your thoughts.
